goal_src/game.gp

```diff
@@ -1876,7 +1876,7 @@
 
 ;; Custom or Modified Code
 (goal-src "pc/pckernel-h.gc" "dma-disasm")
-(goal-src "pc/pckernel.gc" "gsound")
+(goal-src "pc/pckernel.gc" "settings")
```

Comment:
Cool, I wanted to put the file as early as possible so that we don't run into issues where the pc stuff is used before it's properly loaded.